nodes/stampzilla-server/handlers/insecurewebsockethandler.go

```diff
@@ -79,6 +80,8 @@ func (wsh *insecureWebsocketHandler) Connect(s interfaces.MelodySession, r *http
 		UUID:    wsh.Config.UUID,
 		TLSPort: wsh.Config.TLSPort,
 		Port:    wsh.Config.Port,
+		Init:    wsh.Store.CountAdmins() < 1,
+		Login:   helpers.IsPrivateIP(r.RemoteAddr),
```

Comment:
Login == true means what?

Reply:
Its probably a bad name but its used in the webui to know if it should show the login box or an error message saying "Only client certificates are accepted as login when connecting from the internet"